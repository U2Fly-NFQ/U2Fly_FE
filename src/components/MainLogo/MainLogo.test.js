import renderer from 'react-test-renderer'
import MainLogo from '.'
import { I18nextProvider } from 'react-i18next'
import i18n from '../../translations'
import { BrowserRouter as Router } from 'react-router-dom'

describe('MainLogo test', () => {
  it('MainLogo component renders correctly', () => {
    const tree = renderer
      .create(
        <I18nextProvider i18n={i18n}>
          <Router>
            <MainLogo />
          </Router>
        </I18nextProvider>
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
