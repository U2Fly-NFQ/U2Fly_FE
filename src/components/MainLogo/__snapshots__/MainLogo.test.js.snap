// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainLogo search test MainLogo search component renders correctly 1`] = `
<div
  style={Object {}}
>
  <img
    alt="Main logo"
    onClick={[Function]}
    src="logo.png"
    style={
      Object {
        "cursor": "pointer",
        "maxHeight": "64px",
        "objectFit": "contain",
        "transition": "all 0.2s linear",
        "width": "160px",
      }
    }
  />
</div>
`;

exports[`MainLogo test MainLogo component renders correctly 1`] = `
<div
  style={Object {}}
>
  <img
    alt="Main logo"
    onClick={[Function]}
    src="logo.png"
    style={
      Object {
        "cursor": "pointer",
        "maxHeight": "64px",
        "objectFit": "contain",
        "transition": "all 0.2s linear",
        "width": "160px",
      }
    }
  />
</div>
`;
